ui/src/components/song/SongCard.tsx

```diff
@@ -43,104 +59,98 @@ const SongCard = (props: SongCardProps) => {
     filterData?.display?.originalKey ?? true,
     filterData?.display?.year ?? true,
     filterData?.display?.code ?? true,
+    filterData?.display?.timeSignature ?? true,
   ];
-  const fieldData = [themes, timeSignature, tempo, originalKey, year, code];
-  const CardFields = ['Themes', 'Time Signature', 'Tempo', 'Original Key', 'Year', 'Code'];
-
-  // state for the popover, to detect whether mouse is hovering or not
-  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
-
-  const handlePopoverOpen = (event: React.MouseEvent<HTMLElement>) => {
-    setAnchorEl(event.currentTarget);
-  };
+  const fieldData = [themes, tempo, originalKey, year, code, timeSignature];
+  const navigate = useNavigate();
 
-  const handlePopoverClose = () => {
-    setAnchorEl(null);
-  };
-
-  const open = Boolean(anchorEl);
+  // state for the modal
+  const [modalOpen, setModalOpen] = useState(false);
+  const handleOpen = () => setModalOpen(true);
+  const handleClose = () => setModalOpen(false);
 
   return (
-    <>
-      <Container
-        sx={{
-          borderRadius: '4px',
-          background: '#FAFAFA',
-          padding: '10px',
-        }}
-      >
-        <Stack direction="row" display="flex" justifyContent="space-between">
-          <Stack>
-            <Typography variant="h2" color={'primary.main'}>
-              {title}
-            </Typography>
-            <Typography variant="body1">{artist}</Typography>
-          </Stack>
-          {filterData?.display?.lyricsPreview === true ||
-          filterData?.display?.lyricsPreview === undefined ? (
-            <>
-              <Box
-                aria-owns={open ? 'mouse-over-popover' : undefined}
-                aria-haspopup="true"
-                onMouseEnter={handlePopoverOpen}
-                onMouseLeave={handlePopoverClose}
-                sx={{ height: '30px', width: '30px' }}
-              >
-                <VisibilityIcon sx={{ color: 'primary.main' }} />
-              </Box>
-              <Popover
-                id="mouse-over-popover"
-                sx={{
-                  pointerEvents: 'none',
-                }}
-                open={open}
-                anchorEl={anchorEl}
-                anchorOrigin={{
-                  vertical: 'bottom',
-                  horizontal: 'left',
-                }}
-                transformOrigin={{
-                  vertical: 'top',
-                  horizontal: 'left',
-                }}
-                onClose={handlePopoverClose}
-                disableRestoreFocus
-              >
-                <Typography sx={{ p: 1 }}>{lyricsPreview}</Typography>
-              </Popover>
-            </>
-          ) : null}
+    <Container
+      sx={{
+        borderRadius: '4px',
+        background: 'secondary.lighter',
```

Comment:
I dont think the backgroun is working properly need to change it to specifically backgroundColor
```suggestion
        backgroundColor: 'secondary.lighter',
```